{"name": "Попытка получения заказов не авторизованнным пользователем", "status": "passed", "start": 1719213957647, "stop": 1719213957761, "uuid": "edb41bbd-964f-48cc-a3b4-ad5d71302b0f", "historyId": "f6e4b2a532d86a043fed1a8cb1e3455b", "testCaseId": "f6e4b2a532d86a043fed1a8cb1e3455b", "fullName": "tests.test_get_orders.TestGetOrders#test_get_orders_not_auth_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_orders"}, {"name": "subSuite", "value": "TestGetOrders"}, {"name": "host", "value": "DESKTOP-RJ2RHPG"}, {"name": "thread", "value": "6948-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_orders"}]}